dev-python/six/six-1.12.0.recipe

```diff
@@ -41,6 +41,10 @@ BUILD_PREREQUIRES="$BUILD_PREREQUIRES
 	cmd:python$pythonVersion"
 done
 
+REPLACES_python="
+	python_six
+	"
```

Comment:
Any reason to add a REPLACES_python? If none, please remove as always.

Reply:
Haven't been checking up on these, I used the same base as for the other recipes, as far as I can see they build ok for all python versions?

Reply:
Did python_six ever exist?